src/app/pages/quickMaps/pages/components/sideNavContent/sideNavContent.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Can the path of this just be "src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.ts"?

Does it need to be in "pages/components" rather than just "components"?

Reply:
Obvioulsy @JohannDascalescu doesn't think so because he's merged it!